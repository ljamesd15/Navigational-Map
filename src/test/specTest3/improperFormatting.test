# Improper formatting tests.

LoadGraph improperSpace improperSpacing.tsv

LoadGraph improperToken improperTokensPerLine.tsv

LoadGraph improperFile thisFileDoesNotExist.tsv