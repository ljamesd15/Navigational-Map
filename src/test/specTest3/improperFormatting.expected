# Improper formatting tests.

graph improperSpacing.tsv could not be loaded due to formatting issues.

graph improperTokensPerLine.tsv could not be loaded due to formatting issues.

thisFileDoesNotExist.tsv does not exist.